### PR TITLE
Fixed the problem that comment reply cannot be displayed in the front end

### LINKED ISSUE #2951: 前端无法显示评论回复
### 是什么版本出现了此问题？

2.0.1

### 使用的什么数据库？

H2

### 使用的哪种方式部署？

Docker

### 在线站点地址

_No response_

### 发生了什么？

前台无法显示开启审核后的评论。

### 相关日志输出

_No response_

### 附加信息

解决： [解决方案来源](https://github.com/halo-sigs/plugin-comment-widget/issues/3#issuecomment-1351387539)
只需要将 `run.halo.app.content.comment.ReplyServiceImpl `的 `reply.getSpec().setHidden(!reply.getSpec().getApproved())`; 修改为初始值 `false` 即可

https://github.com/halo-dev/halo/blob/7a2f5d0a99575e70942dac7a13600650a2ae2b1f/src/main/java/run/halo/app/content/comment/ReplyServiceImpl.java#L65

刚开始使用halo，修改还是各位大佬修改

### PR DESCRIPTION
**Pull request by minliacom**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
/kind api-change

#### Which issue(s) this PR fixes:

Fixes #2951 

#### Does this PR introduce a user-facing change?

```release-note
修复开启评论审核的情况下无法显示评论回复的问题
```

### CODE REVIEW

File: src/main/java/run/halo/app/content/comment/ReplyServiceImpl.java [hunk 1]
```diff
@@ -62,7 +62,8 @@ public Mono<Reply> create(String commentName, Reply reply) {
                         }
                         reply.getSpec().setApproved(
                             Boolean.FALSE.equals(commentSetting.getRequireReviewForNew()));
-                        reply.getSpec().setHidden(!reply.getSpec().getApproved());
+                        //updated for issue 2951 121522
```
Comment (ruibaby) [inline src/main/java/run/halo/app/content/comment/ReplyServiceImpl.java:65]:
```suggestion
                        // fix https://github.com/halo-dev/halo/issues/2951
```

**Review by ruibaby — CHANGES_REQUESTED** [review]

**Review by guqing — COMMENTED** [review]
/approve

**Review by ruibaby — COMMENTED** [review]
/lgtm

**Review by ruibaby — APPROVED** [review]

**Pull request merged**